packages/angular_devkit/build_angular/src/dev-server/index.ts

```diff
@@ -197,7 +197,7 @@ export class DevServerBuilder implements Builder<DevServerBuilderSchema> {
       port: options.port,
       headers: { 'Access-Control-Allow-Origin': '*' },
       historyApiFallback: {
```

Comment:
If no index is provided shouldn't this be disabled?

Reply:
The default value for it is `index.html` so that seemed appropriate.  This also enables a use case wherein a developer copies a pre-generated index file via an asset configuration.